src/bot/__init__.py

```
@@ -0,0 +1,37 @@
+    # During testing or when database service is not available
+    user_service = None
+
+__all__ = [
+    "LifeWeeksBot",
+    "user_service",
+    "BaseHandler",
+    "SettingsHandler",
+    "StartHandler",
+    "UnknownHandler",
+    "WeeksHandler",
+]
```

Ruff (0.13.1)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)